### PR TITLE
Homer updates

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

Updates in README and Ansible following https://github.com/EGI-Federation/fedcloud-dashboard/pull/56 and https://github.com/EGI-Federation/fedcloud-dashboard/pull/55

---

**Related issue :**

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
Even though there is this:

https://github.com/EGI-Federation/fedcloud-dashboard/blob/fedc6c6e9057089a5f8ce79114ed5ffbb1755d69/dashboard/update_config.py#L21

Locally when I build the dashboard I get all OpenStack dashboards in 1 column instead of 3.

Do you get the same?

**Comment by sebastian-luna-valero** [comment]
As discussed after testing, here is the required change to display multiple columns: https://github.com/EGI-Federation/fedcloud-dashboard/pull/57/commits/2f710048d7f5854ffc586a3a6ba170ac2d2e4d0e

File: deployment/playbook.yaml [hunk 1]
```diff
@@ -26,12 +26,6 @@
       version: "{{ git_ref }}"
       dest: /fedcloud-dashboard
 
-  - name: env file
```
Comment (enolfc) [inline deployment/playbook.yaml:29]: good catch :)

**Review by enolfc — APPROVED** [review]
LGTM

**Pull request merged**